### ISSUE TITLE
error: unrecognized arguments: --arch_instance_dae res_ho_attn

### ISSUE THREAD

**Issue by wangherr** [NVlabs/LSGM#5]
When training cifar10:
~~~
    CUDA_VISIBLE_DEVICES=***** python train_vada.py --fid_dir $FID_STATS_DIR --data $DATA_DIR/cifar10 --root $CHECKPOINT_DIR \
    --save $EXPR_ID/lsgm3 --vae_checkpoint $EXPR_ID/vae3/checkpoint.pt --train_vae --apply_sqrt2_res --fir \
    --cont_kl_anneal --dae_arch ncsnpp --embedding_scale 1000 --dataset cifar10 --learning_rate_dae 1e-4 \
    --learning_rate_min_dae 1e-4 --epochs 1875 --dropout 0.2 --batch_size 16 --num_channels_dae 512 --num_scales_dae 3 \
    --num_cell_per_scale_dae 8 --sde_type geometric_sde --sigma2_min 3e-5 --sigma2_max 0.999 --sigma2_0 3e-5 \
    --weight_decay_norm_dae 1e-2 --weight_decay_norm_vae 1e-2 --time_eps 0.0 --train_ode_eps 1e-6 --eval_ode_eps 1e-6 \
    --train_ode_solver_tol 1e-5 --eval_ode_solver_tol 1e-5 --iw_sample_p ll_uniform --iw_sample_q reweight_p_samples \
    --arch_instance_dae res_ho_attn --num_process_per_node 8 --use_se
~~~

I met the error:
~~~
error: unrecognized arguments: --arch_instance_dae res_ho_attn
~~~

using the pycharm's "Find in Files", I only found  it in `readme.md`
![图片](https://user-images.githubusercontent.com/68674487/164838505-28e510f4-5af7-4ee8-a59f-e5a85d67e39c.png)

but, when I use pretrained weight `CIFAR-10 NLL/checkpoint_nll.p` to run `evaluate_vada.py` , there is `arch_instance_dae=res_ho_attn` in Namespace
![图片](https://user-images.githubusercontent.com/68674487/164838859-86a74b99-18e5-4f91-9ea1-6424e4dad06f.png)

------------------------------------------------------------------------------------------------------------------------

To train cifar10, Could I just delete the `--arch_instance_dae res_ho_attn` in the command or waiting？

**Comment by arash-vahdat** [comment]
`--arch_instance_dae res_ho_attn` is a legacy argument that should not change the flow of our scripts. Please remove it from your commands.

Can you confirm that you could successfully load our released pretrained checkpoints? 

Sorry for the confusion, I will update our commands on the readme.

**Comment by wangherr** [comment]
Yes, I could load the released pretrained checkpoints and get the good results.
